lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -909,17 +909,17 @@ def wait_for_down(self, vm):
             # Stateless snapshot may be already removed:
             if snap_stateless:
                 wait(
```

Comment:
Can you please add here comment:

```
We nned to wait for Active snapshot ID, to be removed as it's current stateless snapshot. Then we need to wait for staless snapshot ID to be read, for use, because it will become active snapshot.
```